### PR TITLE
Update Python.org.xml

### PR DESCRIPTION
**Pull request by koops76**

### CODE REVIEW

File: src/chrome/content/rules/Python.org.xml [hunk 1]
```diff
@@ -1,68 +1,76 @@
...
 
-	<securecookie host="^(bugs|cheeseshop|discuss|openbadges\.es|jobs|mail|packages|packaging|pl|planet|speed|staging|staging2|status|testpypi|warehouse|warehouse-staging|)\.python\.org$" name=".+" />
+	<securecookie host=".+" name=".+" />
 
-	<rule from="^http:"
-		to="https:" />
+	<rule from="^http:" to="https:" />
 
+	<!-- HTTP 403 at / -->
+	<exclusion pattern="^http://svn\.python\.org/$" />
```
Comment (J0WI) [inline src/chrome/content/rules/Python.org.xml]: Please add a valid test url/keep the old one

File: src/chrome/content/rules/Python.org.xml [hunk 2]
```diff
@@ -1,68 +1,76 @@
...
+		- hg
 
-	No working URL known:
-		* front.python.org
+	Incomplete certificate chain:
+		- hg.es
+		- lists.es
 
-	Refused:
-		* buildbot.python.org
```
Comment (J0WI) [inline src/chrome/content/rules/Python.org.xml]: please keep this comment

File: src/chrome/content/rules/Python.org.xml (hunk 1 above)
Comment (koops76) [reply src/chrome/content/rules/Python.org.xml]: The test URL is already here, implicitly. Each target has an implicit test URL of `http://${domain}/`.

File: src/chrome/content/rules/Python.org.xml (hunk 2 above)
Comment (koops76) [reply src/chrome/content/rules/Python.org.xml]: @J0WI Done.

File: src/chrome/content/rules/Python.org.xml (hunk 1 above)
Comment (J0WI) [reply src/chrome/content/rules/Python.org.xml]: Yes, but you just excluded this implicit test url. Now we have no tests for `svn.python.org` at all.

Comment (koops76) [reply src/chrome/content/rules/Python.org.xml]: @J0WI Done.

File: src/chrome/content/rules/Python.org.xml (hunk 2 above)
Comment (koops76) [reply src/chrome/content/rules/Python.org.xml]: @J0WI Can you review as many PRs by me as possible? I won't be available for any corrections for 2 weeks.

**Pull request merged**